web_src/js/components/DashboardRepoList.js

```diff
@@ -138,7 +139,7 @@ function initVueComponents() {
 
     mounted() {
       this.changeReposFilter(this.reposFilter);
-      $(this.$el).find('.tooltip').popup();
+      initTooltip($(this.$el).find('.tooltip')[0]);
```

Comment:
Maybe here is a chance to get rid of jQuery (just for this line). `$(this.$el).find('.tooltip')[0]` seems quite strange in the Vue context.

Reply:
Hmm yes, good opportunity.

Reply:
Done. This `$el` is btw not a jQuery element but some internal vue reference to the root element it seems. Extra confusing.